### PR TITLE
Update CHANGELOG.md for 1.23 strong mode changes

### PR DESCRIPTION
**Pull request by jmesserly**

### CODE REVIEW

**Comment by jmesserly** [comment]
Did this as a pull request for easier previewing/editing.

**Review by leafpetersen — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,56 @@
   characters to the console on Windows. Calls to `Stdout.add*()` behave as
   before.
 
+### Strong Mode
```
Comment (mit-mit) [inline CHANGELOG.md]: I think the last few releases has had these under the top `###Language` section, with the individual changes then calling out if they are strong mode specific or not?

Comment (mit-mit) [inline CHANGELOG.md]: But maybe this is different as these are semantic changes to existing language constructs?

Comment (jmesserly) [inline CHANGELOG.md]: ah that's a fair point! thoughts @leafpetersen ?

Comment (leafpetersen) [inline CHANGELOG.md]: SGTM, I had the same thought.

**Comment by jmesserly** [comment]
Chatted with Leaf, he suggested I merge this then he'll combine with his other 1.23 language changes. Thanks all!

**Pull request merged**